src/constants/constants.ts

```diff
@@ -188,6 +188,14 @@ export interface RMGParam {
     theme: Theme;
     line_name: Name;
     current_stn_idx: keyof StationDict;
+    /**
+     * Display the station number or not.
+     */
+    showStationNumber: boolean;
+    /**
+     * Display the station number in railmap or not.
+     */
+    showStationNumberRailmap: boolean;
```

Comment:
Like the resolve of Disney Icon. There're some person reason in it (For my Minecraft server), sorry.

Reply:
If you need the station number on only one canvas, why not download one canvas and switch the station number button and then download another canvas?

Reply:
Because now numbers in railmap were replaced, but at Xinjiangwancheng - Hangzhong Road/Hongqiao Railway Station the station numbers on runin also there.(Xinjiangwancheng removed.)